pkg/image/anaconda_container_installer.go

```diff
@@ -76,11 +76,6 @@ func (img *AnacondaContainerInstaller) InstantiateManifest(m *manifest.Manifest,
 	anacondaPipeline.ExtraPackages = img.ExtraBasePackages.Include
 	anacondaPipeline.ExcludePackages = img.ExtraBasePackages.Exclude
 	anacondaPipeline.ExtraRepos = img.ExtraBasePackages.Repositories
-	anacondaPipeline.InteractiveDefaultsKickstart = &kickstart.Options{
```

Comment:
`InteractiveDefaultsKickstart` isn't used anymore?

Reply:
Not for the container installer.  There we always use our own, at `/osbuild.ks`.
We use the interactive defaults for the live and image installer sometimes though.